### PR TITLE
[12.0][ADD] - Add new module Product Contract Transmit Method

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: product_contract_transmit_method/README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
+================================
+Product Contract Transmit Method
```
Comment (bodi000) [inline product_contract_transmit_method/README.rst]: I think the name is misleading, as it is a sale order to contract link, not dealing with product. The `product_contract` family deals precisely with products linked to contracts. I suggest `sale_contract_transmit_method` or `contract_sale_transmit_method` to align it to the naming of other similar modules.

File: product_contract_transmit_method/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    @api.multi
+    def _prepare_contract_value(self, contract_template):
+        contract_values = super()._prepare_contract_value(contract_template)
+        contract_values["transmit_method_id"] = self.transmit_method_id.id
```
Comment (bodi000) [inline product_contract_transmit_method/models/sale_order.py]: usually the simple quotas are used for fields, like so `'transmit_method_id'`

**Review by bodi000 — CHANGES_REQUESTED** [review]
IMO some small changes are needed. thank you

**Comment by sbejaoui** [comment]
cc/ @len-foss

File: product_contract_transmit_method/README.rst (hunk 1 above)
Comment (len-foss) [reply product_contract_transmit_method/README.rst]: +1

File: product_contract_transmit_method/models/sale_order.py (hunk 2 above)
Comment (len-foss) [reply product_contract_transmit_method/models/sale_order.py]: It's 13.0 (Black) coding convention (easier to forward-port).

**Review by len-foss — APPROVED** [review]
I agree that the name should reflect better the module. Besides that, all good.

**Comment by sbejaoui** [comment]
@gva-acsone , can you review this PR please

**Review by gva-acsone — APPROVED** [review]
LGTM Functional test

**Pull request closed without merging**